src/webgpu/web_platform/canvas/getCurrentTexture.spec.ts

```diff
@@ -263,21 +286,35 @@ g.test('resize')
     t.expect(currentTexture.height === ctx.canvas.height);
     t.expect(prevTexture.width === 4);
     t.expect(prevTexture.height === 2);
-    prevTexture = currentTexture;
 
     // Ensure that texture contents are transparent black.
     t.expectSingleColor(currentTexture, currentTexture.format, {
       size: [currentTexture.width, currentTexture.height, 1],
       exp: { R: 0, G: 0, B: 0, A: 0 },
     });
 
-    // Simply setting the canvas width and height values to their current values should not trigger
-    // a change in the texture.
-    ctx.canvas.width = 4;
-    ctx.canvas.height = 4;
```

Comment:
This feels like a valid test scenario that we don't want to lose? Either this counts as being modified and we should verify that it destroys the current texture in the process or it doesn't and we should ensure that the current texture is retained.

Reply:
This was moved to like 311

```
    // Setting the canvas width and height values to their current values should
    // still trigger a change in the texture.
    const { width, height } = ctx.canvas;
    ctx.canvas.width = width;
    ctx.canvas.height = height;
```

Reply:
Apologies for missing that!